sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -104,404 +56,7 @@ const client = new TextAnalyticsClient("<endpoint>", new DefaultAzureCredential(
-
-const client = new TextAnalyticsClient("<endpoint>", new AzureKeyCredential("<API key>"));
-
-const documents = [
-  "Microsoft was founded by Bill Gates and Paul Allen.",
-  "The employee's SSN is 555-55-5555.",
-  "Easter Island, a Chilean territory, is a remote volcanic island in Polynesia.",
-  "I use Azure Functions to develop my product."
-];
-
-async function main() {
-  const actions = {
-    recognizeEntitiesActions: [{ modelVersion: "latest" }],
-    recognizePiiEntitiesActions: [{ modelVersion: "latest" }],
-    extractKeyPhrasesActions: [{ modelVersion: "latest" }]
-  };
-  const poller = await client.beginAnalyzeBatchActions(documents, actions);
-  const resultPages = await poller.pollUntilDone();
-  for await (const page of resultPages) {
-    const keyPhrasesAction = page.extractKeyPhrasesResults[0];
-    if (!keyPhrasesAction.error) {
-      for (const doc of keyPhrasesAction.results) {
-        console.log(`- Document ${doc.id}`);
-        if (!doc.error) {
-          console.log("\tKey phrases:");
-          for (const phrase of doc.keyPhrases) {
-            console.log(`\t- ${phrase}`);
-          }
-        } else {
-          console.error("\tError:", doc.error);
-        }
-      }
-    }
-
-    const entitiesAction = page.recognizeEntitiesResults[0];
-    if (!entitiesAction.error) {
-      for (const doc of entitiesAction.results) {
-        console.log(`- Document ${doc.id}`);
-        if (!doc.error) {
-          console.log("\tEntities:");
-          for (const entity of doc.entities) {
-            console.log(`\t- Entity ${entity.text} of type ${entity.category}`);
-          }
-        } else {
-          console.error("\tError:", doc.error);
-        }
-      }
-    }
-
-    const piiEntitiesAction = page.recognizePiiEntitiesResults[0];
-    if (!piiEntitiesAction.error) {
-      for (const doc of piiEntitiesAction.results) {
-        console.log(`- Document ${doc.id}`);
-        if (!doc.error) {
-          console.log("\tPii Entities:");
-          for (const entity of doc.entities) {
-            console.log(`\t- Entity ${entity.text} of type ${entity.category}`);
-          }
-        } else {
-          console.error("\tError:", doc.error);
-        }
-      }
-    }
-  }
-}
-
-main();
-```
-
-## Known Issues
-
-- `beginAnalyzeHealthcare` is still in gated preview and can not be used with AAD credentials. For more information, see (the Text Analytics for Health documentation)[https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-for-health?tabs=ner#request-access-to-the-public-preview].
-- At time of this SDK release, the `modelVersion` option to `beginAnalyzeHealthcareEntities` is ignored by the service. The service always processes the operation using the "latest" model.
+`TextAnalyticsClient` is the primary interface for developers using the Azure Text Analytics Client library. It provides asynchronous methods to access a specific use of Text Analytics.
```

Comment:
> It provides asynchronous methods to access a specific use of Text Analytics

I believe this can be improved...
How about.

> Explore the methods on this client object to understand the different features of the Text Analytics service that you can access.

Reply:
Sure. I was just using what is currently being used in TA's readme. Perhaps suggestions in this PR will need to be applied to Track 2 packages READMEs as well.